libs/blocks/preflight/panels/seo.js

```diff
@@ -237,6 +237,16 @@ async function checkLinks() {
   const baseOpts = { method: 'POST', headers: { 'Content-Type': 'application/json' } };
   const badResults = [];
 
+  links.forEach((link) => {
+    if (link.dataset?.httpLink) {
```

Comment:
```suggestion
    if (link.href.includes('http:')) {
```

Reply:
I resolved the above conversation with a comment, please let me know if that is not satisfactory @narcis-radu. Hoping to resolve this one as well.

Reply:
Resolving this conversation. I believe the linked discussion in the above comment illustrates why we need to do this check in utils, i.e. before milo link transformation.